copycat/api/src/main/java/org/apache/kafka/copycat/connector/CopycatRecord.java

```diff
@@ -57,10 +62,18 @@ public Object getKey() {
         return key;
     }
 
+    public Schema getKeySchema() {
```

Comment:
Ah yes, one day I will dig through IntelliJ's settings to figure out how to get it to autogenerate these with the right names...

I've updated these and tried to fix up a bunch of other cases where I used auto-generated helpers. Hopefully this doesn't make this more annoying to review.